builtin/logical/rabbitmq/path_config_connection.go

```diff
@@ -0,0 +1,109 @@
+package rabbitmq
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+	"github.com/michaelklishin/rabbit-hole"
+)
+
+func pathConfigConnection(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config/connection",
+		Fields: map[string]*framework.FieldSchema{
+			"uri": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "RabbitMQ Management URI",
+			},
+			"username": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Username of a RabbitMQ management administrator",
+			},
+			"password": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Password of the provided RabbitMQ management user",
+			},
```

Comment:
There is a need for `verify_connection` option (refer postgresql backend).
This option makes the `connection_uri` verification below, toggle-able, during the configuration.